setup.py

```diff
@@ -39,6 +39,7 @@
         'taca.subcommands': [
             'storage = taca.storage.cli:storage',
             'analysis = taca.analysis.cli:analysis',
+            'bioinfo = taca.utils.cli:bioinfo',
```

Comment:
@Galithil  - How about `bioinfo-deliveries`? `run-status`?

@vezzi - whilst displayed on the new deliveries page, it's also displayed under the 'Bioinfo Analysis' tab, so `bioinfo` is still kind of relevant..